src/esm_runscripts/filedicts.py

```diff
@@ -73,6 +72,23 @@ def inner_method(self, *args, **kwargs):
     return inner_method
 
 
+def _fname_has_date_stamp_info(fname, date, reqs=["%Y", "%m", "%d"]):
```

Comment:
Missing docstrings

Reply:
...and type hints. I'll take care of that.